package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "console-monorepo",
```

Comment:
```suggestion
  "name": "console-sdk-monorepo",
```

This is not the monorepo of the console, it is the console sdk

Reply:
changed